### PR TITLE
Improve product sample

### PR DESCRIPTION
**Pull request by kul3r4**
* Add Google Store Logo

* Use blue for buttons and links

* Add price

This is also related to #255

### CODE REVIEW

**Comment by sebastianbenz** [comment]
Nice! Two small nits:

* give the related articles cards the same margin as the headline
* align the buy button with the title

File: src/50_Sample_AMPs/Product_Sample.html [hunk 1]
```diff
@@ -15,266 +15,286 @@
...
+            <h2>Nexus 6P</h2>
+            <a href="/shopping-cart-sample.html" class="button button-primary call-to-action seller-button">Buy</a>
+        </p>
 
-    <!-- #### Image Gallery -->
-    <!-- The `amp-carousel` works very well for product image galleries. Learn more about `amp-carousel` [here](/advanced/image_galleries_with_amp-carousel/). -->
+        <p id="product-description">
+            A 5.7-inch, high-resolution WQHD AMOLED display and front-facing stereo speakers — all housed in a contoured aluminum body.
+        </p>
+        <h5 class="price-description">From 399.00$</h5>
```
Comment (sebastianbenz) [inline src/50_Sample_AMPs/Product_Sample.html]: Prices are often dynamic which can be a problem if the page is accessed via the AMP Cache. Could we use amp-list to include a dynamic price?

**Comment by sebastianbenz** [comment]
rename file s/Product_Sample.html/Product/

**Comment by kul3r4** [comment]
@sebastianbenz PTAL

File: src/50_Sample_AMPs/Product.html [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+        <div class="seller-logo">
+            <amp-img src="/img/google_store.png" width="163" height="35"></amp-img>
+        </div>
+
+        <!-- This button adds the item to the shopping cart. Pressing this button should lead to a the shopping cart page on your normal website and the cart should be prefilled with the current item. -->
+        <div id="product-header">
+            <h2>Nexus 6P</h2>
+            <a href="/shopping-cart-sample.html" class="button button-primary call-to-action seller-button">Buy</a>
+        </div>
+        <!-- Use [amp-list](/components/amp-list/) to ensure the pages shows the latest price. -->
```
Comment (sebastianbenz) [inline src/50_Sample_AMPs/Product.html]: add: "if it is loaded via the Google AMP Cache."

File: src/50_Sample_AMPs/Product.html [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+        <amp-list width=auto height=100 layout=fixed-height
+          src="/json/product.json">
```
Comment (sebastianbenz) [inline src/50_Sample_AMPs/Product.html]: use /json/product.json?RANDOM for cache busting

**Comment by kul3r4** [comment]
Suggestions added, @sebastianbenz PTAL

**Comment by sebastianbenz** [comment]
Great! LGTM

**Pull request merged**